src/components/Header/Header.tsx

```diff
@@ -52,8 +55,13 @@ export const Header = () => {
             <ToolbarGroup variant="filter-group">
               {user && (
                 <ToolbarItem>
-                  <Button className="chr-c-button-allservices" component={(props) => <ChromeLink {...props} href="/AllServices" />}>
-                    All Services
+                  <Button className="chr-c-button-masthead" component={(props) => <ChromeLink {...props} href="/AllServices" />}>
+                    <CloudIcon />
+                    Services
+                  </Button>
+                  <Button className="chr-c-button-masthead disabled" component={(props) => <ChromeLink {...props} href="#" />}>
```

Comment:
This should be hidden behind a feature flag. We don't want to show this page/button before its ready.

Reply:
@Hyperkid123 Kat would like to show it as a teaser. That's why I styled it as a disabled link.

Reply:
Ah, I was not aware of that. OK then